src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public static void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message printed should be more informative. It could indicate that the type of machine working is a 'Truck'. For example, 'Truck is starting its work'.